tachyon/math/base/groups.h

```diff
@@ -96,23 +97,23 @@ class AdditiveGroup : public AdditiveSemigroup<G> {
     }
   }
 
-  // Subtraction in place: a -= b
+  // Subtraction in place:
+  //   1) a -= b if subtraction is supported.
+  //   2) a += (-b) otherwise
   template <
       typename G2,
-      std::enable_if_t<internal::SupportsSubInPlace<G, G2>::value>* = nullptr>
-  constexpr G& operator-=(const G2& other) {
-    G* g = static_cast<G*>(this);
-    return g->SubInPlace(other);
-  }
-
-  // Subtraction in place: a += (-b)
-  template <
-      typename G2,
-      std::enable_if_t<!internal::SupportsSubInPlace<G, G2>::value &&
+      std::enable_if_t<internal::SupportsSubInPlace<G, G2>::value ||
```

Comment:
Can you fix the commit message? I think you have united `operator-=()` not `operator*=()`.